app/controllers/articles_controller.rb

```diff
@@ -0,0 +1,68 @@
+class ArticlesController < ApplicationController
+  load_and_authorize_resource
+  before_action :set_article, only: %i[ show edit update destroy ]
+
+  def index
+    @articles = Article.all
+  end
+
+  def show
+    @articles = Article.find(params[:id])
+  end
+
+  def new
+    @article = Article.new
+  end
+
+  def create
+    @article = Article.new(article_params)
+
+    # El metodo current_user, lo provee device a todos los controladores y vistas
+    # para acceder a el usuario logueado
+    @article.owner = current_user
+    respond_to do |format|
```

Comment:
@sansuaza Please explain the **respond_to** and response **format** woks?

Reply:
respond_to is use to indicate the types of format to be returned (html and json in this case)